packages/swingset-liveslots/src/vatstore-usage.md

```diff
@@ -50,42 +50,44 @@ Each time the kind constructor is called, a new "baseref" is allocated for the c
+Each instance of a virtual object stores state in a vatstore key indexed by the baseref. If `o+d12/1:0` and `o+d12/1:1` are the facet vrefs for a cohort whose baseref is `o+d12/1`, the cohort's shared state will be stored by the virtual object manager in `vom.o+d12/1` as a JSON-serialized record. The keys of this record are property names: if the Kind uses `state.prop1`, the record will have a key named `prop1`. For each property, the value is a `{ body: string, slots: any[] }` capdata record with string-valued slot items (and unlike the treatment of capdata in many other parts of the system, it is not serialized independently of the enclosing structure).
 
 * `v6.vs.vom.o+d12/1` : `{"booleanProp":{"body":"true","slots":[]},"arrayProp":{"body":"[]","slots":[]}}`
 
 In the refcounting portion of the vatstore (`vom.rc.${baseref}`), you will see baserefs:
 
 * `v6.vs.vom.rc.o+10`: the count of virtualized references to plain Remotable `o+10` (held in RAM)
-* `v6.vs.vom.rc.o+d12/1`: the count of references to any member of the cohort for the first instance of Kind `o+d12`
+* `v6.vs.vom.rc.o+d12/1`: the count of references to any member of the cohort for the first instance of durable Kind `o+d12`
   * This Kind might be single-facet or multi-facet.
   * References to distinct facets of the same cohort are counted independently. For example, if one object references both the first and second facets (`o+d12/1:0` and `o+d12/1:1`), it accounts for two increments to this value.
 
 In the export-status portion of the vatstore (`vom.es.${baseref}`), you will see baserefs, and any facets are tracked in the value, not the key:
 
 * `v6.vs.vom.es.o+10` records the export status for plain Remotable `o+10`
   * value `r`: the plain Remotable has been exported and is "reachable" by the kernel
-  * value `s`: the Remotable was exported, the kernel dropped it, and is still "recognizable" by the kernel ("s" for "see")
+  * value `s`: the Remotable was exported, the kernel dropped it, and is still "recognizable" by the kernel ("s" for "see", refer to [Garbage Collection in SwingSet](../../SwingSet/docs/garbage-collection.md) for details)
   * If the kernel can neither reach nor recognize the export, the vatstore key will be missing entirely.
-* `v6.vs.vom.es.o+d12/1` records the export status for all facets of virtual object `o+d12/1`
+* `v6.vs.vom.es.o+d12/1` records the export status for all facets of the first instance of durable Kind `o+d12`
```

Comment:
please add "(aka the virtual object whose vref is `o+d12/1`)" or similar, to remind the reader that the `o+d12/1` suffix in the original vatstore key is a virtual object first, and a composition of KindID plus instance/sequence number second

also, to be technical, this is referring to a durable object, not a merely-virtual object, so maybe my recommended edit should s/virtual/durable/